typescript/forms.d.ts

```diff
@@ -891,22 +891,32 @@ export interface ReceiptRandomComponent {
   lowercase: boolean
 }
 
+export interface ReceiptAutoIncrementComponent {
+  /** The type of receipt component */
+  type: 'auto_increment'
```

Comment:
why snake case? too much Nylas?

Reply:
we normally make constants like this snake case, however, the other `type` properties are all lower case...I panicked!

Reply:
If they're uppercase yah.